srcpkgs/wire-desktop/template

```diff
@@ -1,21 +1,19 @@
 # Template file for 'wire-desktop'
 pkgname=wire-desktop
 version=3.24.2939
-revision=1
+revision=2
 wrksrc="${pkgname}-linux-${version}"
 hostmakedepends="nodejs yarn git"
+depends="electron10"
 short_desc="Wire for desktop"
 maintainer="Young Jin Park <youngjinpark20@gmail.com>"
 license="GPL-3.0-or-later"
 homepage="https://wire.com"
 distfiles="https://github.com/wireapp/wire-desktop/archive/linux/${version}.tar.gz"
 checksum=69e70d39286f29b2fb186da30ec828be0a30329c457e829111b863223ed64127
-nocross=yes
 
 case "$XBPS_TARGET_MACHINE" in
-	*-musl) broken="glibc artefacts are installed" ;;
-	i686*) broken="yarn" ;;
-	ppc*) broken="electron" ;;
+	x86_64-musl) broken="Electron on musl is unreliable" ;;
```

Comment:
IMO, not necessary. Element works nearly perfectly for me, and anything electron working on musl is a bonus already, so we should leave people the option of trying their luck.